web/src/lib/components/ui/tags-multiple-select/tags-multiple-select.svelte

```diff
@@ -2,17 +2,14 @@
   import * as Select from '$lib/elements/ui/select';
   import type { Tag } from '$lib/sdk/fetch-client';
   import type { Selected } from 'bits-ui';
-  import { onMount } from 'svelte';
 
   export let tags: Tag[];
   let selectedTags: Selected<string>[] = [];
-  export let selectedTagIds: unknown[];
-  export let entityTags: Tag[];
-
-  onMount(() => {
-    selectedTags = entityTags.map((tag) => ({ label: tag.name, value: tag.id }));
-    selectedTagIds = selectedTags.map((tag) => tag.value);
-  });
+  export let selectedTagIds: string[];
+  selectedTags = tags
```

Comment:
Two minor things: (1) exports should be in one batch at the top. (2) you can just immediately assign `tags.filter().map()` to the `selectedTags` field. Also, I think this should be reactive, so `$: selectedTags = tags.filter().map()`, right?